redfish-core/lib/log_services.hpp

```diff
@@ -1004,6 +1061,10 @@ inline void createDump(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
         dumpPath = "/xyz/openbmc_project/dump/bmc";
     }
 
+    createDumpParams.emplace_back(
```

Comment:
The clientIP is the one which needs to be stored as GeneratorId.

Reply:
Done